integral_view/templates/format_disks.html

```diff
@@ -1,25 +1,29 @@
-{% extends 'logged_in_base.html' %}
+{% extends 'storage_base.html' %}
 
 
 {%block tab_header %}
-  Format Disk
+  Format disk
 {%endblock%}
 
 {% block inside_content %}
 
-  <form id="create_form" name="create_form" action="/format_complete" method="GET">
+  <form id="create_form" name="create_form" action="/format_disk/" method="POST">
+  {%csrf_token%}
     <input name="path" type="hidden" value="{{ form.path.value }}">
+    <input name="disk_id" type="hidden" value="{{ disk_id  }}" >
+
     <table class="table" style="width:800px">
+
       <tr>
         <th> USB id :</th>
```

Comment:
This should read "Disk ID" and not "USB id" because it may be used for disks also.